src/sites/twitch-twilight/modules/featured_follow.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+// ============================================================================
+// Featured Follow
+// ============================================================================
+
+import Module from 'utilities/module';
+import {createElement as e} from 'utilities/dom';
+
+import FEATURED_QUERY from './featured_follow_query.gql';
+
+import FEATURED_FOLLOW from './featured_follow_follow.gql';
+import FEATURED_UNFOLLOW from './featured_follow_unfollow.gql';
+
+const TWITCH_URL = /^(?:https?:\/\/)?(?:www\.)?twitch\.tv\/([A-Za-z0-9_]+)/i;
+
+export default class FeaturedFollow extends Module {
+	constructor(...args) {
+		super(...args);
+
+		this.should_enable = true;
+
+		this.inject('site');
+		this.inject('site.fine');
+		this.inject('site.apollo');
+		this.inject('i18n');
+		this.inject('metadata');
+		this.inject('settings');
+		this.inject('socket');
+		this.inject('site.router');
+
+		this.inject('chat');
+
+		this.settings.add('metadata.featured-follow', {
+			default: true,
+
+			ui: {
+				path: 'Channel > Metadata >> Player',
+				title: 'Featured Follow',
+				description: 'Show a featured follow button with the currently featured users.',
+				component: 'setting-check-box'
+			},
+
+			changed: () => {
+				this.metadata.updateMetadata('following');
+			}
+		});
+
+		this.metadata.definitions.following = {
+			order: 150,
+			button: true,
+
+			popup: async (data, tip) => {
+				const vue = this.resolve('vue'),
+					_featured_follow_vue = import(/* webpackChunkName: "featured-follow" */ './featured-follow.vue'),
+					_follows = this.getFollowsForLogin(data.channel.login);
+
+				const [, featured_follows_vue, follows] = await Promise.all([vue.enable(), _featured_follow_vue, _follows]);
+
+				this._featured_follow_tip = tip;
+				tip.element.classList.remove('tw-pd-1');
+				tip.element.classList.add('tw-balloon--lg');
+				vue.component('featured-follow', featured_follows_vue.default);
+				return this.buildFeaturedFollowMenu(vue, data.channel.login, follows);
+			},
+
+			label: data => {
+				if (!this.settings.get('metadata.featured-follow')) return '';
```

Comment:
Why are there two separate if statements that just return an empty string? And you may as well return null